### DISCUSSION TITLE
App update adoption (iOS & Android)

### DISCUSSION THREAD

**Discussion by kavithachandramohan-okta** [MobileNativeFoundation/discussions#132] [General]
Hi All, 

Are there any known metrics or benchmarks we should strive for with regards to app updates adoption? I was thinking more on the lines of what would be considered a good update adoption rate which could reduce the burden of investing too much into maintaining older versions. Going by firebase analytics, our latest version adoption rate is 85%, trying to understand where we fall on a scale of worst to excellent. 

Thanks
Kavitha

**Comment by robpridham** [comment]
For this 85% figure, when was your release rolled out? I don't think you can have a meaningful metric unless you introduce time into it. Most people get updated very quickly but it can take a few weeks before everyone moves over.

There is a separate question in here though and it's this: if I have users unnecessarily stuck (i.e. their device is compatible with the latest release) on old versions for whatever reason, how often & after how long should I force them to update?

**Reply by 8vqjx80b** [reply]
Suomessa ja kansainvälisesti 85 prosentin päivitysnopeus on itse asiassa erinomainen tulos. Yleensä 70–80 prosenttia pidetään hyvänä, ja siitä yli menevät lukemat [pelikaani-kasino-fi](https://pelikaani-kasino-fi.com/) ovat jo erinomaisia. Jos pääsette yli 90 prosentin, se alkaa olla huipputasoa. Vanhempien versioiden tuki on toki helpompi lopettaa, kun käyttäjämäärät laskevat.
